src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.model.dto.request;
+
+public class UserRequestDto {
+    private String email;
+    private String password;
```

Comment:
You can use annotations to validate this parameters, but of course it's optional:)